package-lock.json

```diff
@@ -26828,6 +26862,7 @@
         "prettier": "^2.2.1"
       }
     },
+    "packages/react-hook": {},
```

Comment:
Hmm. Doing an `npm install` doesn't show any differences... but you're right, this is an error and there's other parts in the package-lock which have a similar error, e.g.:
```
    "packages/jsapi-components/node_modules/@deephaven/react-hooks": {
      "resolved": "packages/log",
      "link": true
    },
```
We fixed a similar issue with #1247 , #1248... it's worrying that `npm install` isn't identifying any issues though.
Might just delete package-lock.json and regenerate, since it does not look right...

Reply:
I must have screwed something up with it while merging previous changes. In any case, I just copied the `package.json` from `main` and ran an `npm install` again, and now it looks correct.